### ISSUE TITLE
Datagrid: Not possible to tab past a single row action trigger

### ISSUE THREAD

**Issue by amellnik** [vmware/clarity#5136]
## Describe the accessibility issue

On a Datagrid with single row actions (like https://clarity.design/documentation/datagrid/single-action) it's not possible to tab past the first action trigger.  Hitting tab once opens the action menu which traps focus, and while it can be closed with `ESC` or similar this returns the focus to the trigger, and the same thing happens if the user tries to tab past it again.  

Discovered while looking at #4756.  

## WCAG Guideline

https://www.w3.org/TR/WCAG21/#keyboard

## How to reproduce

Reproduceable at https://clarity.design/documentation/datagrid/single-action.

Try to tab through the page.  

## Expected behavior

The menu should only open if the user hits space or enter with the trigger focused (or possibly the menu should not trap focus, but I don't think that's a good solution).  

## Versions

**App**

- Angular: [e.g. 6]
- Node: [e.g. 8.10.0]
- Clarity: [e.g. 0.12.5]

**Device:**

- Type: Desktop
- OS: Ubuntu
- Browser Chrome
- Version 85
- Screen Reader: N/A

**Comment by Jinnie** [comment]
Hi @amellnik 

I am not able to reproduce it currently against the website. Is it possible that it got fixed in the meantime?
I am doing the following:
- Open https://clarity.design/documentation/datagrid/single-action (tested with v3 and v4)
- Tab into the datagrid.
- Tabbing moves between action menu trigger icons on the rows (was it supposed to fail here?)
- Open an action menu
  - Now focus is trapped and I move between items in the menu
  - Hit ESC
- Back at the action menu trigger icon.
- Tabbing more moves me to the next trigger icon.

Am I trying to reproduce it wrong?

**Comment by amellnik** [comment]
This now appears fixed -- before tabbing to the trigger was enough to open the action menu, but now I have to hit enter or space to open it, which allows me to tab past it.  Closing, and I'll also check the other related issues and see if they have also been fixed.

**Comment by Jinnie** [comment]
Thank you, Alex
